pandas/tests/api/test_api.py

```diff
@@ -251,3 +251,16 @@ def test_categorical_move(self):
 
         with tm.assert_produces_warning(FutureWarning):
             from pandas.core.categorical import CategoricalDtype  # noqa
+
+    @pytest.mark.parametrize("attr", [
+        "Grouper", "groupby", "BinGrouper", "Grouper", "_GroupBy", "GroupBy",
+        "SeriesGroupBy", "_pipe_template", "PanelGroupBy", "Grouping",
+        "SpecificationError", "DataError", "generate_bins_generic"])
+    def test_groupby_move(self, attr):
```

Comment:
This would be straightforward to combine with the method above if we wanted one test accepting a module / list of objects to test the deprecation for